qml/MainPage.qml

```diff
@@ -13,38 +13,104 @@ Page {
             return new RegExp('');
         }
     }
+    property bool searchMode: false
 
     header: PageHeader {
+        id: defaultHeader
         title: i18n.tr("Lomiri Touch Logs")
         flickable: scrollView.flickableItem
+    }
+
+    states: [
+        State {
+          id: defaultState
```

Comment:
I would rather only define non-default states here (might cause confusion since Qt has the concept of a default state, so now we have two)

Reply:
just tried before and there where an issue with trailingActions not reseted to its original values iirc. Will give a try again

Reply:
yes confirmed, trailingActions are not reset...

Reply:
Oh my, UITK has some serious issues